server/routes/signup.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const router = express.Router();
+const bcrypt = require("bcrypt");
+const saltRounds = 10;
+const User = require("../models/User");
+
+router.post("/:email/:password", (req, res, next) => {
+  const { email, password } = req.params;
+  let epCheck = false;
+  if (email.includes("@") && password.length > 6 && (epCheck = true)) {
+    bcrypt.hash(password, saltRounds, (err, hash) => {
+      const addUser = async () => {
+        const user1 = new User({
+          email: email,
+          password: hash,
+        });
+        await user1
```

Comment:
You don't need this await if you're using a .then() block, this also means it doesn't need to be in an async function